pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -90,6 +90,11 @@ public enum TransformFunctionType {
   // date type conversion functions
   CAST("cast"),
 
+  // object type
+  ARRAY_TO_MV("arrayToMV",
```

Comment:
IMO this is confusing name. the data type is already MV running an ARRAY_TO_MV is a bit weird. 
should we named it `USE_AS_MV`? and we can say that MV columns are by default `USE_AS_ARRAY`

Reply:
`USE_AS_MV` might be confusing, since we repurposed `MV` as `ARRAY` in v2, `ARRAY_TO_MV` might be more explicit.